extensions/resteasy-classic/rest-client/deployment/src/test/java/io/quarkus/restclient/configuration/RestClientOverrideRuntimeConfigTest.java

```diff
@@ -41,9 +41,6 @@ void overrideConfig() {
         assertTrue(specifiedDefaultValues.isPresent());
         assertTrue(specifiedDefaultValues.get().getPropertyNames()
                 .contains("io.quarkus.restclient.configuration.EchoClient/mp-rest/url"));
-        // This config key comes from the interceptor and it is recorded with an empty value. This allow the user to still override using the original key
-        assertEquals("", specifiedDefaultValues.get()
-                .getValue("quarkus.rest-client.\"io.quarkus.restclient.configuration.EchoClient\".url"));
```

Comment:
@radcortez FYI - since the "url" property is no longer processed as part of any ConfigRoot, it's not going to be present in the "specifiedDefaultValues" source, so I need to remove this bit. Let me know if you want me to modify this test differently.

Reply:
It's ok. This was because of https://github.com/quarkusio/quarkus/pull/20590 but with this PR is not required anymore.